moonshot.p8

```diff
@@ -42,19 +41,46 @@ function init_player()
   h=8,
   dx=0,
   dy=0,
+  acch=1,
+  accv=1,
+  frct=0.3,
   max_dx=2,
   max_dy=3,
   flp=false,
   sp=1,
   update=function()
+
    --apply gravity
    plr.dy=mid(
     -plr.max_dy,
     plr.dy+g,
     plr.max_dy
    )
-   
-   
+
+   -- Apply frct to dx
```

Comment:
Please avoid capitals, the built-in editor displays them smaller than regular letters.